Log form `install-arm-uwp-dbg-out.log`:
````
D:\buildtrees\cubeb\src\9e009830ea-4cc11e5bd0.clean\src\cubeb_winmm.c(102): error C2061: syntax error: identifier 'WAVEHDR'
D:\buildtrees\cubeb\src\9e009830ea-4cc11e5bd0.clean\src\cubeb_winmm.c(109): error C2061: syntax error: identifier 'waveout'
````
You can get the latest error log [here](https://dev.azure.com/vcpkg/public/_build/results?buildId=77342&view=artifacts&pathAsName=false&type=publishedArtifacts)
[install-arm-uwp-dbg-out.log](https://github.com/microsoft/vcpkg/files/9476668/install-arm-uwp-dbg-out.log)

--- a/ports/cubeb/portfile.cmake
+++ b/ports/cubeb/portfile.cmake
@@ -1,0 +1,23 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO mozilla/cubeb
+    REF dc511c6b3597b6384d28949285b9289e009830ea
+    SHA512 a4ccd3f0a156db4e2e75a8d231e95a08d555390571551cb3e92c71cdee46dc74dc66b5272fda4b5f1f083b92672b360152cefd38be242f238fe802acc1ea17e9
+    HEAD_REF master
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH ${SOURCE_PATH}
+    OPTIONS
+        -DUSE_SANITIZERS=OFF
+        -DBUILD_TESTS=OFF
+        -DBUILD_TOOLS=OFF
+)
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/cubeb)
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
+    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/bin" "${CURRENT_PACKAGES_DIR}/debug/bin")
+endif()

--- a/ports/cubeb/vcpkg.json
+++ b/ports/cubeb/vcpkg.json
@@ -1,0 +1,18 @@
+{
+  "name": "cubeb",
+  "version-date": "2022-07-22",
+  "description": "Cross platform audio library",
+  "homepage": "https://github.com/mozilla/cubeb",
+  "license": "ISC",
+  "supports": "!uwp",
+  "dependencies": [
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ]
+}

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1760,6 +1760,10 @@
       "baseline": "1.16.0",
       "port-version": 0
     },
+    "cubeb": {
+      "baseline": "2022-07-22",
+      "port-version": 0
+    },
     "cuda": {
       "baseline": "10.1",
       "port-version": 11

--- a/versions/c-/cubeb.json
+++ b/versions/c-/cubeb.json
@@ -1,0 +1,9 @@
+{
+  "versions": [
+    {
+      "git-tree": "7d7979530500923609dae0ce99dc4a63fea13056",
+      "version-date": "2022-07-22",
+      "port-version": 0
+    }
+  ]
+}

